images/base/Dockerfile

```diff
@@ -1,12 +1,12 @@
 #
 # This is the base image from which all OpenShift Origin images inherit. Only packages
-# common to all downstream images should be here. Depends on Centos 7.2+.
+# common to all downstream images should be here. Depends on Centos 7.2+
 #
 # The standard name for this image is openshift/origin-base
 #
-FROM centos:centos7
+FROM registry.centos.org/centos/centos:7
```

Comment:
Does this have better uptime and bandwidth than Dockerhub?

Reply:
The registry is meant to always be up, and be the official source of the centos images.
@bamachrn if you want to add ^

Reply:
yes, also we are working for making this as a default registry for centos docker.

Reply:
@smarterclayton ^

Reply:
When you say default registry... what do you mean?

Reply:
We mean that it will be added by default, to `/etc/sysconfig/docker` on future centos releases.
@bamachrn ^

Reply:
@smarterclayton ^

Reply:
@smarterclayton The registry is meant to always be up, even if the Container Pipeline Service goes down. There are backup registries that get synced regularly and a geo located mirroring that is being worked on as well @bamachrn

Reply:
@kbsingh ^